content/providers/03-community-providers/94-langdb.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: LangDB
+description: Learn how to use LangDB with the AI SDK
+---
+
+# LangDB
+
+[LangDB](https://langdb.ai) is a high-performance enterprise AI gateway built in Rust, designed to govern, secure, and optimize AI traffic.
+
+## Installation
```

Comment:
Please follow the structure / section names of the other community providers (e.g. `Setup`)